app/services/content_metadata_generator.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+# Builds the contentMetadata xml
+class ContentMetadataGenerator
+  # @param [String] druid the object identifier
+  # @param [Array<String>] file_names a list of full paths to the files
+  def self.generate(druid:, file_names:)
+    new(file_names: file_names, druid: druid).generate
+  end
+
+  def initialize(file_names:, druid:)
+    @file_names = file_names
+    @druid = druid
+  end
+
+  def generate
+    Assembly::ContentMetadata.create_content_metadata(cm_params)
+  end
+
+  private
+
+  attr_reader :file_names, :druid
+
+  def cm_params
+    {
+      druid: druid,
+      objects: object_files,
+      file_attributes: file_attributes,
+      add_file_attributes: true,
+      add_exif: false,
+      bundle: :filename,
+      style: :simple_book
```

Comment:
If this is a temporary hardcode, can you please call out with a comment?

Reply:
I don't know the answer to that without feedback from Andrew.